recipes/mapdamage2/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: mapdamage2
+  version: "2.0.6"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: 2.0.6.tar.gz
+  url: https://github.com/ginolhac/mapDamage/archive/2.0.6.tar.gz
+  md5: f14e1ca52d68fd9c7e42e1f4e6f40f8c
+
+requirements:
+  build:
+    - python >=2.7,<3
```

Comment:
please depend on GCC and LLVM as described here: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc